src/main/java/uk/nhs/tis/trainee/notifications/migration/DeleteUnusedPlacementUpdateHistory.java

```diff
@@ -39,7 +39,7 @@
  * Delete unused placement-update history records.
  */
 @Slf4j
-@ChangeUnit(id = "deleteUnusedPlacementUpdateHistory", order = "3", runAlways = true)
+@ChangeUnit(id = "deleteUnusedPlacementUpdateHistory", order = "3")
```

Comment:
Turn off the ongoing clearing of the history in anticipation of actually sending mails